build.gradle

```
@@ -3,175 +3,46 @@ buildscript {
 }
 
 plugins {
-    id 'application'
-    id 'com.diffplug.spotless' version "${spotlessVersion}"
-    id 'antlr'
-    id 'com.github.ben-manes.versions' version "${benmanesVersion}"
-    // use: gradlew dependencyUpdates -Drevision=release
+    id "com.diffplug.spotless" version "6.23.3"
```

```suggestion
    id 'com.diffplug.spotless' version "${spotlessVersion}"
```
versionen über die dependencies.gradle holen, damit die autom. verwaltet werden können